src/python_testing/TC_RVCRUNM_2_2.py

```diff
@@ -0,0 +1,202 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import chip.clusters as Clusters
+from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main, type_matches
+from mobly import asserts
+
+# This test requires several additional command line arguments.
+# Run the test with
+# --int-arg PIXIT_ENDPOINT:<endpoint> PIXIT.RVCRUNM.MODE_A:<mode id> PIXIT.RVCRUNM.MODE_B:<mode id>
+
+
+def error_enum_to_text(error_eum):
+    if error_eum == 0:
+        return "Success(0x00)"
+    elif error_eum == 0x1:
+        return "UnsupportedMode(0x01)"
+    elif error_eum == 0x2:
+        return "GenericFailure(0x02)"
+    elif error_eum == 0x3:
```

Comment:
This is going to fall out of date with the generated code at some point if you add more enum values.

You can use .name to get the name of the enum for printing. You have the number already. 
```
try:
  return f'{Clusters.RvcRunMode.Enums.ModeTag(error_enum).name} (0x{error_enum:02x})
except <uh...can't remember what gets throw here, but some error>:
  # convert to the generic error
```